exercises/practice/protein-translation/protein-translation.spec.coffee

```diff
@@ -134,4 +134,4 @@ describe 'ProteinTranslation', ->
   xit "Non-existing codon can't translate", ->
     expect ->
       ProteinTranslation.proteins("AAA")
-    .toThrow "Invalid codon"
\ No newline at end of file
+    .toThrow new Error("Invalid codon")
```

Comment:
As suggested in https://github.com/exercism/coffeescript/pull/265#issuecomment-1919713374, maybe use:

```suggestion
    .toThrowError('Invalid codon')
```

Reply:
The checks failed with this change, looks like this functions has two parameters.